sys/net/gnrc/Makefile.dep

```diff
@@ -441,6 +441,25 @@ ifneq (,$(filter gnrc_ipv6_nib,$(USEMODULE)))
   USEMODULE += random
 endif
 
+#This checks if the option is being set via Kconfig or CFLAGS
+ifneq (,$(or $(CONFIG_GNRC_IPV6_STABLE_PRIVACY),$(filter -DCONFIG_GNRC_IPV6_STABLE_PRIVACY=1,$(CFLAGS))))
+  # Set another macro that is needed for this option.
+
+  ##prepare value
+  stable_privacy_secret_key != python3 -c "import secrets; print(','.join([f'0x{byte:02x}' for byte in secrets.token_bytes(16)]))"
+  #Python >= 3.6, because:
+  #- 3.6 https://docs.python.org/3/library/secrets.html
+  #- 3.6 https://docs.python.org/3/reference/lexical_analysis.html#f-strings
+    #replaceable: f'0x{byte:02x}' == '0x' + format(byte, '02x')
```

Comment:
Good you checked, but given 3.7 is already EoL for good, the comments are probably unwarranted.